packages/esm-patient-registration-app/src/index.ts

```diff
@@ -58,7 +58,6 @@ function setupOpenMRS() {
           savePatientForm: FormManager.savePatientFormOffline,
           isOffline: true,
         },
```

Comment:
I cannot comment at that location, but there is this piece of code above which is now obsolete and can be removed:
![grafik](https://user-images.githubusercontent.com/30902964/166914293-a977803f-76d0-4ed6-8457-df49f75d088f.png)

Reply:
Oh yes, totally forgot about it.
I'll just remove this and check for more redundant code places.